shared/src/fixtures/arrow.rs

```diff
@@ -170,23 +170,23 @@ pub fn primitive_record_batch() -> Result<RecordBatch> {
             // No Float16Array implemented in the arrow lib.
             Arc::new(Float32Array::from(vec![1.0, -1.0, 0.0])),
             Arc::new(Float64Array::from(vec![1.0, -1.0, 0.0])),
-            Arc::new(TimestampSecondArray::from(vec![
-                Some(0),
-                Some(0),
-                Some(1627849445),
-            ])),
+            // Arc::new(TimestampSecondArray::from(vec![
```

Comment:
If you left these commented out on purpose here, could you leave comments explaining why they're commented out?

Reply:
done